Mathlib/SetTheory/Nimber/Basic.lean

```diff
@@ -54,13 +46,14 @@ noncomputable section
 
 /-- A type synonym for ordinals with natural addition and multiplication. -/
 def Nimber : Type _ :=
-  Ordinal deriving Zero, Inhabited, One, WellFoundedRelation
+  Ordinal deriving Zero, Inhabited, One, Nontrivial, WellFoundedRelation
 
 instance Nimber.linearOrder : LinearOrder Nimber := {Ordinal.linearOrder with}
 instance Nimber.succOrder : SuccOrder Nimber := {Ordinal.instSuccOrder with}
 instance Nimber.orderBot : OrderBot Nimber := {Ordinal.orderBot with}
 instance Nimber.noMaxOrder : NoMaxOrder Nimber := {Ordinal.noMaxOrder with}
 instance Nimber.zeroLEOneClass : ZeroLEOneClass Nimber := {Ordinal.zeroLEOneClass with}
+instance Nimber.NeZero.one : NeZero (1 : Nimber) := Ordinal.NeZero.one
```

Comment:
```suggestion
instance Nimber.instNeZeroOne : NeZero (1 : Nimber) := Ordinal.NeZero.one
```

Reply:
If that's the convention, then the ordinal instances should be renamed too: #19064